docs/faq.rst

```diff
@@ -814,6 +814,30 @@ computation at runtime. For example:
 For more information on runtime callbacks and examples of their use,
 see `External callbacks in JAX`_.
 
+Why do some CUDA libraries fail to load/initialize?
+---------------------------------------------------
+
+JAX ultimately relies on :code:`dlopen` to load runtime libraries, which searches
```

Comment:
I'll note this is not 100% true: because we set `RPATH`, we will prefer the pip-installed libraries over `LD_LIBRARY_PATH`. You might mention that.